modules/seqexec/server/src/main/scala/seqexec/server/tcs/TcsController.scala

```diff
@@ -215,7 +215,7 @@ object TcsController {
 
   final case class InstrumentOffset(p: Angle@@OffsetP, q: Angle@@OffsetQ) {
     def toFocalPlaneOffset(iaa: Angle): FocalPlaneOffset = FocalPlaneOffset(
-      tag[OffsetX]((-p * iaa.cos - q * iaa.sin) / FOCAL_PLANE_SCALE),
+      tag[OffsetX](((p * -1 * iaa.cos) - q * iaa.sin) / FOCAL_PLANE_SCALE),
```

Comment:
This is fairly ugly. Scala 2.13.3 forbids calls to methods that take parens without them. In this case we are calling squant's `unary_-()` method and it won't allow it
We maybe able to get a PR into squants to solve it